conan/tools/gnu/pkgconfigdeps/pc_files_creator.py

```diff
@@ -33,15 +33,15 @@ def _get_aliases_pc_files_and_content(conanfile, dep, aliases_info, root_pc_file
     return pc_files
 
 
-def _get_components_pc_files_and_content(conanfile, dep, components_info):
+def _get_components_pc_files_and_content(conanfile, dep, package_info, components_info):
     """
     Get the PC files and content for dependency's components
     """
     pc_files = {}
     # Loop through all the package's components
     for pc_info_component in components_info:
         # Get the *.pc file content for each component
-        description = "Conan component: %s" % pc_info_component.name
+        description = "Conan component: %s-%s" % (package_info.name, pc_info_component.name)
```

Comment:
Was this a bug? why this change?

Reply:
No, it wasn't. Old PkgConfig was showing `[pkg_name]-[component_name]` and I realized that `PkgConfigDeps` is not doing that right now. It's only to keep the same logic.